tests/automated_benchmarking.py

```diff
@@ -10,6 +11,9 @@
 GITHUB_API_PR_URL = "https://api.github.com/repos/facebook/zstd/pulls?state=open"
 GITHUB_URL_TEMPLATE = "https://github.com/{}/zstd"
 MASTER_BUILD = {"user": "facebook", "branch": "dev", "hash": None}
+
+# check to see if there are any new PRs every minute
+MAX_API_CALL_FREQUENCY_SEC = 60
```

Comment:
👍 

minor : ideally, this should be a parameter selectable on command line.
Starting with a constant is still fine, it can later be converted into a nice default.